spec/lib/classroom/scopes_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe Classroom::Scopes do
```

Comment:
Could we add a test in here that will fail if the ordering the different roles by `scopes.size` changes and invalidates the logic we're using to make sure we don't downgrade. That way that dependency is codified.